addons/web_editor/controllers/main.py

```diff
@@ -762,3 +762,22 @@ def bus_broadcast(self, model_name, field_name, res_id, bus_data):
     @http.route('/web_editor/tests', type='http', auth="user")
     def test_suite(self, mod=None, **kwargs):
         return request.render('web_editor.tests')
+
+    @http.route("/web_editor/wysiwyg_iframe", type='http', auth='none')
```

Comment:
![image](https://github.com/odoo/odoo/assets/30048408/0580189d-8216-4604-aa19-c126eee296b7)

Go to runbot, mass mailing, click on a draft mailing -> Crash

Reply:
See https://discord.com/channels/678381219515465750/687338152372994236/1164177225730502658 for more details

Reply:
After discussion with @ged-odoo. It has been decided to abort this PR. The main goal of this PR is to remove getBundle totally from the codebase and only use the controller in webclient.py. We thought that there was only one JS file and one CSS file generated by bundle. After discussions with XDO, we learned that was not always the case. So ... this is not possible to remove getBundle from assets for now.